tools/docker/builder/openwrt/toolchains/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:16.04
+
+RUN dpkg --add-architecture i386
```

Comment:
Can you add a comment for this command? What does it do and what is it needed for?

Reply:
This command adds ability to install packages for i386 architecture.